packages/ccd-js-gen/package.json

```diff
@@ -12,7 +12,7 @@
         "build-dev": "yarn build",
         "clean": "rm -r lib",
         "lint": "eslint src/** bin/** --cache --ext .ts,.js --max-warnings 0",
-        "lint-fix": "yarn --silent lint --fix; exit 0",
+        "lint-fix": "yarn lint --fix; exit 0",
```

Comment:
Why is this changed?

Reply:
It caused this error:

```Unknown Syntax Error: Command not found; did you mean one of: ...```

I kept the syntax as we have it in other repos now e.g.:
https://github.com/Concordium/concordium-dapp-examples/blob/main/dapp-template/frontend/package.json#L10